ge/views.py

```diff
@@ -100,6 +101,21 @@ def edit_librarydata(request: HttpRequest, item_id: int) -> HttpResponse:
     return render(request, "ge/edit_item.html", {"form": form})
 
 
+@login_required(login_url="/login/")
+def add_fund(request: HttpRequest) -> HttpResponse:
+    if request.method == "POST":
+        form = LibraryDataEditForm(request.POST)
+        if form.is_valid():
+            new_fund = form.save()
+            messages.success(request, "Fund added successfully.")
+            return redirect(
+                reverse("edit_librarydata", kwargs={"item_id": new_fund.pk})
+            )
+    else:
+        form = LibraryDataEditForm()
+    return render(request, "ge/add_fund.html", {"form": form})
```

Comment:
As part of the refactoring for consistency, please rename the `edit_item.html` template to match the other changes - probably change to `edit_fund.html`.